python/surf/devices/silabs/_Si5345.py

```diff
@@ -2590,6 +2748,14 @@ def __init__(self,
             hidden      = simpleDisply,
         ))
 
+class Si5345PageA(pr.Device):
+    def __init__(self,       
+            name         = "PageA",
+            description  = "",
```

Comment:
There's no need to include `description` here if you aren't going to give it a default. `""` is already the default.

Reply:
Fixed in 50cac9c